assets/js/data/cart/selectors.js

```diff
@@ -147,11 +137,18 @@ export const isItemQuantityPending = ( state, cartItemKey ) => {
  * @param {Object} state The current state.
  * @return {boolean} are shipping rates loading.
  */
-export const areShippingRatesLoading = createRegistrySelector(
-	( select ) => ( state ) => {
-		return !! (
-			state.metaData.updatingShipping ||
-			select( STORE_KEY ).isResolving( 'getCartData', [] )
-		);
-	}
-);
```

Comment:
I think I originally suggested `areShippingRatesLoading` considering whether cart data is loading.  However I _think_ that was before the shipping address refactor when cart loading state could be triggered by shipping rate update state.  However, we need a _distinct_ loading state here for loading shipping rates so I reverted to only return that value. Testing seemed to be okay with this change.